src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1030,8 +1030,7 @@ namespace winrt::TerminalApp::implementation
                 const auto& profileGuid = focusedTab->GetFocusedProfile();
                 if (profileGuid.has_value())
                 {
-                    const auto settingsImpl{ winrt::get_self<implementation::CascadiaSettings>(_settings) };
-                    const auto settings = settingsImpl->BuildSettings(profileGuid.value());
+                    const auto settings{ winrt::make<TerminalSettings>(_settings, profileGuid.value(), *_bindings) };
```

Comment:
how is this different from BuildSettings?

Reply:
| Function | Params |
|--|--|
| `Constructor` | CascadiaSettings, **guid**, IKeyBindings |
| `BuildSettings` | CascadiaSettings, **NewTerminalArgs**, IKeyBindings |

Before, they were both `BuildSettings`. But the second `BuildSettings` needs to return a tuple so we can't just make that another constructor :(